docs/frontend/external-bus.md

```diff
@@ -155,3 +157,19 @@ Payload structure:
 
 }
 ```
+
+### Write Tag `tag/write`
+
+Available in: Home Assistant 0.115
+Type: `tag/write`
+Direction: frontend to external app
+Expect answer: no
```

Comment:
i can imagine a flow where you'd want to know it's complete, and even if it was successfully written or not. is it worth adding to the spec now?

Reply:
Yes we should add it as we can't make breaking changes. Wouldn't error handling be handled in the native UI though? (Including retry)

Reply:
User-facing error and such should be native, but I can imagine you may e.g. open a dialog waiting for the write and would want to close or refresh it once the write is complete.

Reply:
I can't think of anything right now, but yeah maybe better to already make it have a response, which is just empty for now. We can add more info later. 

CC @JBassett

Reply:
That will require a bit of refactor for us to support but should be fine.